lib/conrad/collector.rb

```diff
@@ -156,13 +179,23 @@ def record_events_as_batch
 
     def record_individual_events
       events.each do |event|
-        emitter.call(event)
+        begin
+          emitter.call(event)
+        rescue StandardError => e
+          write_log(:error, e)
```

Comment:
What's the thinking behind having a wrapper around the standard ruby logger interface? Can we default to
```
module Conrad
  attr_accessor :logger

  self.logger = Logger.new(STDOUT)
end
```

and allow users to override it to be whatever they need?

Reply:
:shrug: I mostly didn't want to send things to STDOUT people weren't prepared for and break possible log shipping they have for poorly/unformatted stuff.

Reply:
That's fair. I've also seen `/dev/null` as the default